### PR TITLE
Userdoc/ashrae901 gettingstarteddemofiles update jc

### PR DESCRIPTION
**Pull request by JayatiChhabra**
2 commits 
1 commit includes changes to the demo files user documentation section 
1 commit includes the changes to zip files and images.

### CODE REVIEW

File: content/get_start/demo/demo1.md [hunk 1]
```diff
@@ -1,96 +1,144 @@
...
 
-The proposed model is a typical medium sized office building with roughly `60%` window-to-wall ratio (WWR) and overhangs on its south facade.
-![Geometry Transformation Image](/BEM-for-PRM/get_start/os_engine/images/geometry_demo_screenshot.png?width=400px&align=right&classes=border,alignLeft)
-{{< line_break >}}
+#### Demo 1 shows geometry transformations in a user model
 
-#### Project parameters
 
-First, we will need to check the project parameters in the script.
+The proposed model (as shown in the image) is a typical medium sized office building with roughly `60%` window-to-wall ratio (WWR) and overhangs on its south facade. The `default_hvac_bldg_type` is set to `other nonresidential`, and the window to wall ratio and service water heating system building types are both set according to the related category. The building is assumed to be located in New York City. So its climate zone is `4A`.
```
Comment (weilixu) [inline content/get_start/demo/demo1.md:16]: Add a note that in this demo, we are using a user model as proposed model to simplify the process.

File: content/get_start/demo/demo1.md [hunk 2]
```diff
@@ -1,96 +1,144 @@
...
+    - Default Building Type for HVAC : **_other nonresidential_**
+    - Climate Zone : **_ASHRAE 169-2013-4A_**
+    - Exempt from Rotations : **_FALSE_** (You can select TRUE for your project if the building has rating authority approval that orientation is dictated by site considerations.)
+    - Exempt from Unmet Load Hours Check : **_TRUE_** (TRUE has been selected here only for demo purposes, otherwise it's always recommended to select FALSE until and unless the rating authority has approved allowing the models to exceed the specified.)
+    - Use User Data : **_FALSE_** (FALSE selected since we are not using any user data for this demo.) 
 
-{{< line_break >}}
+5. **Run Simulation:** Go to the **_Run Simulation_** tab and run the  simulation. Once it's completed, go to the *_demo1_* folder in the system. A new folder `geometry_demo_model` would be created containing subfolders for the runs as shown in [Run the measure] section. The `generated_files` folder would contain the output .osm files, reports that should reflect the Output shown [here](http://localhost:1313/BEM-for-PRM/get_start/demo/demo1/#demo-1-results).
```
Comment (weilixu) [inline content/get_start/demo/demo1.md]: You will need to update this link - use full link will fail the production deployment.
Use relative path: `/BEM-for-PRM/get_start/demo/demo1/#demo-1-results`

File: content/get_start/demo/demo1.md [hunk 3]
```diff
@@ -1,96 +1,144 @@
...
+5. **Run Simulation:** Go to the **_Run Simulation_** tab and run the  simulation. Once it's completed, go to the *_demo1_* folder in the system. A new folder `geometry_demo_model` would be created containing subfolders for the runs as shown in [Run the measure] section. The `generated_files` folder would contain the output .osm files, reports that should reflect the Output shown [here](http://localhost:1313/BEM-for-PRM/get_start/demo/demo1/#demo-1-results). 
+
+<!-- Link Run the measure section -->
+
+{{<line_break>}}
+
+##### **B. Using OS SDK with CLI**
+
+1. **Setup Local Directory:** Set up the files from `demo1` folder in a local directory. One such example of the folder structure is shown in [Run the measure](http://localhost:1313/BEM-for-PRM/get_start/os_cli/run_the_measure/) section. 
+2. **Edit OSW File:** Open the .osw file created initially (Refer to [Run the measure](http://localhost:1313/BEM-for-PRM/get_start/os_cli/run_the_measure/) section). Update the project inputs and parameters as follows.
```
Comment (weilixu) [inline content/get_start/demo/demo1.md]: All these absolute link needs to update

**Review by weilixu — COMMENTED** [review]
The absolute link shall be changed to relative link.

File: content/get_start/demo/demo1.md (hunk 1 above)
Comment (JayatiChhabra) [reply content/get_start/demo/demo1.md:16]: Done. Adding it in the general intro page since this is applicable for all 3 demos.

**Review by weilixu — APPROVED** [review]

**Pull request merged**